loadbalancer/algorithm.go

```diff
@@ -206,9 +207,13 @@ func (ch consistentHash) Less(i, j int) bool { return ch[i].hash < ch[j].hash }
 func (ch consistentHash) Swap(i, j int)      { ch[i], ch[j] = ch[j], ch[i] }
 
 func newConsistentHash(endpoints []string) routing.LBAlgorithm {
-	ch := consistentHash(make([]endpointHash, len(endpoints)))
+	const hashesPerEndpoint = 100
```

Comment:
I am wondering is there any reason to make it configurable?..

Reply:
Since this is used when creating the algorithm (not when processing a request) that would mean we need to make it a available as a CLI flag right? It won't be configurable via a routegroup or ingress. I can see value in configuring it per-application via ingress or routegroup, not sure there's much value in doing it via cli arg :thinking:

Reply:
Yeah, algorithms do not support parametrization yet (the similar problem was with `powerOfRandomNChoices` where N is hardcoded to 2). I think we can leave this for future improvement then.